src/index.js

```diff
@@ -33,14 +33,17 @@ const config = require("./config.json")
 const store = createStore(solrReducer);
 
 // The search fields and filterable facets you want
+// Add collapse to set the default displayed fields
+// collapse: true => hidden
+// collapse: false => shown
 const fields = [
-  { label: "All text fields", field: "searchText", type: "text" },
-  { label: "Identifier", field: "id", type: "text" },
-  { label: "Source", field: "source", type: "list-facet", facetSort: "index" },
-  { label: "Context", field: "hasContextCategory", type: "list-facet", facetSort: "count" },
-  { label: "Material", field: "hasMaterialCategory", type: "list-facet", facetSort: "count" },
-  { label: "Specimen", field: "hasSpecimenCategory", type: "list-facet", facetSort: "count" },
-  { label: "Registrant", field: "registrant", type: "list-facet", facetSort: "count" },
+  { label: "All text fields", field: "searchText", type: "text", collapse: false },
+  { label: "Identifier", field: "id", type: "text", collapse: false },
+  { label: "Source", field: "source", type: "list-facet", facetSort: "index", collapse: false },
+  { label: "Context", field: "hasContextCategory", type: "list-facet", facetSort: "count", collapse: false },
+  { label: "Material", field: "hasMaterialCategory", type: "list-facet", facetSort: "count", collapse: false },
+  { label: "Specimen", field: "hasSpecimenCategory", type: "list-facet", facetSort: "count", collapse: false },
+  { label: "Registrant", field: "registrant", type: "list-facet", facetSort: "count", collapse: false },
```

Comment:
Could we default `collapse` to `false` and only require it if we want to set to `true`?  It might make the code easier to read.